save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -82,8 +82,15 @@ class ProjectService(
 
     /**
      * @param organizationName
+     * @return List of the Organization projects
      */
-    fun findByOrganizationName(organizationName: String) = projectRepository.findByOrganizationName(organizationName).let { Flux.fromIterable(it) }
+    fun getByOrganizationName(organizationName: String) = projectRepository.findByOrganizationName(organizationName)
```

Comment:
getByOrganizationName and findByOrganizationName are similar -- can we rename the second as `getAsFluxByOrganizationName`?

Reply:
@Arrgentum `find*` and `get*` methods by convention refer to a single (or absent) result. If you are returning a collection, it's preferrable to name method `findAll*` or `getAll*`. Here you could use `getAllAsFlux*`